lib/index.d.ts

```diff
@@ -437,3 +435,181 @@ declare namespace SwaggerParser {
   }
 
 }
+
+/**
+ * Parses, dereferences, and validates the given Swagger API.
+ * Depending on the options, validation can include JSON Schema validation and/or Swagger Spec validation.
+ *
+ * See https://apitools.dev/swagger-parser/docs/swagger-parser.html#validateapi-options-callback
+ *
+ * @param api An OpenAPI definition, or the file path or URL of an OpenAPI definition. See the `parse` method for more info.
+ * @param options (optional)
+ * @param callback (optional) A callback that will receive the dereferenced OpenAPI definition
+ */
+export function validate(api: string | OpenAPI.Document, callback: SwaggerParser.ApiCallback): void;
```

Comment:
Unfortunately, I had to duplicate the definition. I am not aware of any other way to expose methods both as part of a class, and as something that can be accessed via named import.